index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
 specifically:
         </p>
 
-        <ul>
+        <ol>
           <li>
-<a>Holders</a> receive and store <a>verifiable credentials</a> from
-<a>issuers</a>. <a>Holders</a> might interact with an <a>issuer</a> through an
-agent that the <a>issuer</a> does not need to trust.
+<a>Verifiable Credentials</a> represent statements made by an <a>Issuer</a> in
+a tamper evident and privacy respecting manner.
           </li>
           <li>
-<a>Holders</a> are positioned between <a>issuers</a> and <a>verifiers</a>
-and use <a>verifiable credentials</a> to make <a>verifiable presentations</a>
-to <a>verifiers</a>.
+<a>Holders</a> assemble collections of claims from different issuers into a
+single artifact, a <a>Verifiable Presentation</a>.
           </li>
           <li>
-<a>Holders</a> might interact with <a>verifiers</a> through an agent that
-<a>verifiers</a> do not need to trust; <a>verifiers</a> only need to trust
-<a>issuers</a>.
+<a>Issuers</a>  can issue <a>Verifiable Credentials</a> about any <a>subject</a> .
           </li>
           <li>
-<a>Verifiable credentials</a> are associated with <a>subjects</a>, not
-specific services; <a>holders</a> decide how to aggregate and manage
-<a>verifiable credentials</a> and the <a>verifiable presentations</a>
-associated with them.
+Acting as <a>Issuer</a>, <a>Holder</a>, or <a>Verifier</a> requires neither registration
+nor approval by any authority.
           </li>
           <li>
-<a>Subjects</a> can easily acquire and control their identifiers.
+<a>Verifiable Presentations</a>  allow any <a>Verifier</a> to verify the authenticity
+of claims from any Issuer.
           </li>
           <li>
-<a>Holders</a> control which <a>verifiable credentials</a> to use and when.
+<a>Holders</a> can receive <a>Verifiable Credentials</a> from anyone.
           </li>
           <li>
-<a>Holders</a> can freely choose and change the agents they employ to help
-them manage their <a>verifiable credentials</a> and generate and share
-<a>verifiable presentations</a>.
+<a>Holders</a> can interact with any <a>Issuer</a> and any <a>Verifier</a> through
+any User Agent.
           </li>
           <li>
-<a>Holders</a> can share <a>verifiable presentations</a> that can be verified
-without revealing the identity of the <a>verifier</a> to the <a>issuer</a>.
+<a>Holders</a> can store <a>Verifiable Credentials</a> in any location, without
```

Comment:
True in general, but problematic in practice. For example, if the issuer also provides the revocation list via their systems, and tracks access to the list, then they could tell when the credential is used.

Reply:
I think this is fine, "stored and accessed" without the issuer knowing is different than "used" without the issuer knowing.